packages/protocol/StreamingScreenShotCache.ts

```diff
@@ -43,7 +43,7 @@ export const StreamingScreenShotCache = createStreamingCache<
 
     const paintPoint = findMostRecentPaint(time);
 
-    if (paintPoint) {
+    if (paintPoint && paintPoint.paintHash) {
```

Comment:
Out of curiosity, do you usually opt for this syntax over `paintPoint?.paintHash`?

Reply:
It's subjective, but I think it's more explicit that we're accounting for two cases: no paint found and an empty paint